cincinnati/src/lib.rs

```diff
@@ -1017,6 +1040,45 @@ pub mod testing {
 
         removed_metadata
     }
+
+    /// Replace the digests by versioned tags in the release payload strings.
+    pub fn payload_replace_sha_by_tag_fn(graph: &mut Graph) -> Fallible<()> {
+        graph
+            .iter_releases_mut(|ref mut release| {
+                match release {
+                    Release::Concrete(ref mut release) => {
+                        // replace digest by tag to match expectency
+                        let version = release.version.to_string();
+                        let source_front = release.payload.split('@').next().ok_or_else(|| {
+                            Error::msg(format!("invalid version string {:?}", version))
+                        })?;
+                        release.payload = format!("{}:{}", source_front, version);
+
+                        Ok(())
+                    }
+                    _ => panic!("should not get here"),
+                }
+            })
+            .context("Post-processing the received graph to match the test expectations")
```

Comment:
This message `Post-processing the received graph to match the test expectations` is only specific to tests and does not sound right.